include/cantera/oneD/Flow1D.h

```diff
@@ -510,15 +515,14 @@ class Flow1D : public Domain1D
      * default boundary condition is zero radial velocity (@f$ V @f$) at the left
      * and right boundary.
      *
-     * For argument explanation, see evalContinuity().
+     * For argument explanation, @see evalContinuity().
```

Comment:
Look at the rendered HTML, and you'll see that `@see` doesn't do what you want -- it was fine before. (Here and in several other locations below).